covid_moonshot/analysis/plots.py

```diff
@@ -251,6 +253,67 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[Run], energy_cutoff_kcal: float = 1.0,
+) -> plt.Figure:
+    """
+    Plot table of poorly converging free energy estimates with GEN
+
+    Parameters
+    ----------
+    runs : list of Run
+        Relative free energies (in kT)
+    energy_cutoff_kcal : float
+        Cutoff to consider a result as poorly converged (in kcal/mol)
+
+    Returns
+    -------
+    Figure
+        Figure containing the plot
+
+    """
+
+    complex_phases = [run.analysis.complex_phase for run in runs]
+    job_ids = [run.details.JOBID for run in runs]
```

Comment:
These defs are unused, can they be removed?

Reply:
Ah yes they can, good catch!